posthog/api/feature_flag.py

```diff
@@ -115,6 +115,14 @@ class FeatureFlagSerializer(TaggedItemSerializerMixin, serializers.HyperlinkedMo
     )
     can_edit = serializers.SerializerMethodField()
 
+    CREATED_FROM_CHOICES = ("flags", "experiments", "surveys", "early_access_features", "web_experiments")
```

Comment:
nit: do we have a convention for referring to us as "flags" vs "feature_flags" that we should keep consistent?

Reply:
hmmm gp gp gp.  I'll switch it to `feature_flags`